Controller/WebsiteCommentController.php

```diff
@@ -118,17 +118,31 @@ public function cgetCommentsAction(string $threadId, Request $request): Response
     {
         list($type, $entityId) = $this->getThreadIdParts($threadId);
 
-        $page = $request->get('page');
+        $limit = $request->query->getInt('limit') ?? 10;
+        $offset = $request->query->getInt('offset') ?? 0;
+
+        $pageSize = $request->get('pageSize') ?? 10;
+        if ($pageSize) {
```

Comment:
This is always true.

```suggestion
        $pageSize = $request->get('pageSize');
        if ($pageSize) {
```